### PR TITLE
test: Unify users.controller.test and users.api.test (no-changelog)

### PR DESCRIPTION
**Pull request by RicardoE105**
Groundwork to be able to safely refactor and move the invitation logic to the UserService.

Fixes ADO-1358

### CODE REVIEW

**Comment by RicardoE105** [comment]
Fixes: https://linear.app/n8n/issue/ADO-1358/add-tests-to-post-users-endpoint

**Comment by RicardoE105** [comment]
@netroy Do you know why we have `user.controller.test` and `users.api.test`? I just rewrote the test because I checked in the controller and did not know we had a users.api.test. Should we have only one for tests the that belong to users.controller?

File: packages/cli/test/unit/Helpers.ts [hunk 1]
```diff
@@ -36,3 +37,10 @@ export function mockNodeTypesData(
 		);
 	}, {});
 }
+
+/**
+ * A helper to return the arguments a mocked function was last called with.
+ */
+export const getLastArgsTo = <T>(subject: MockedFunction<any>) => {
```
Comment (netroy) [inline packages/cli/test/unit/Helpers.ts:44]: this doesn't seem to be used

**Review by netroy — CHANGES_REQUESTED** [review]

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.17.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.17.0)

**Pull request merged**